lib/timescale/migration.ex

```diff
@@ -67,9 +67,8 @@ defmodule Timescale.Migration do
   defp opts_to_sql_arg([]), do: ""
 
   defp opts_to_sql_arg(opts) do
-    ", " <>
-      Enum.map(opts, fn {k, v} ->
-        "#{k} => #{v}"
-      end)
+    Enum.map_join(opts, ", ", fn {k, v} ->
```

Comment:
DId Credo suggest this refactor? Nice!

Reply:
This is actually a bug caught by Dialyzer @davydog187! I tested it out locally and sure enough the prior version resulted in an argument error.

Reply:
Amazing!